tests/simulator/refinement/test_2d_10_core.py

```diff
@@ -197,7 +197,7 @@ def main():
     startMPI()
     rando = random.randint(0, int(1e10))
     Simulator(config(L0_diags, {"seed": rando})).run().reset()
-    refinement_boxes = {"L0": {"B0": [(7, 40), (20, 60)]}}
+    refinement_boxes = {"L0": {"B0": [(5, 40), (20, 59)]}}
```

Comment:
there might be a problem in the `test_refine_field.py`

but changing it to this prevents it from appearing :shrug: 

```
  File "/path/to/git/phare/master/tests/simulator/refinement/test_2d_10_core.py", line 206, in <module>
    main()
  File "/path/to/git/phare/master/tests/simulator/refinement/test_2d_10_core.py", line 202, in main
    Simulator(sim, post_advance=post_advance).run()
  File "/path/to/git/phare/master/pyphare/pyphare/simulator/simulator.py", line 164, in run
    self.advance()
  File "/path/to/git/phare/master/pyphare/pyphare/simulator/simulator.py", line 142, in advance
    self.post_advance(self.cpp_sim.currentTime())
  File "/path/to/git/phare/master/tests/simulator/refinement/test_2d_10_core.py", line 179, in post_advance
    test.base_test_field_level_ghosts_via_subcycles_and_coarser_interpolation(
  File "/path/to/git/phare/master/tests/simulator/test_advance.py", line 582, in base_test_field_level_ghosts_via_subcycles_and_coarser_interpolation
    interpolated_fields = refine_time_interpolate(
                          ^^^^^^^^^^^^^^^^^^^^^^^^
  File "/path/to/git/phare/master/tests/amr/data/field/refine/test_refine_field.py", line 399, in refine_time_interpolate
    refine_algo(
  File "/path/to/git/phare/master/tests/amr/data/field/refine/test_refine_field.py", line 98, in refine_electric
    fine_data[1::2, ::2] = coarse_data[:, :]

```

Reply:
what is the error? shape? and why would that be a problem *now*?

Reply:
> what is the error? shape?

shape mismatch, I forgot to copy that part

> and why would that be a problem now?

this is never run on any PR/nightly build currently (we could fix that too)

Reply:
I think we can merge these fixes and deal with that error later if it's not yet run anyway.
Could you make an issue reporting the error so we don't forget?

Reply:
sure